src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -677,7 +677,12 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('workbench.enableExperiments', "Fetches experiments to run from a Microsoft online service."),
 			'default': true,
 			'tags': ['usesOnlineServices']
-		}
+		},
+		'workbench.tasks.quickOpen.history': {
```

Comment:
The setting shouldn't be in the workbench scope. It should be `tasks.quickOpen.history`. Like terminal is top level as well.